tests/test_button.py

```diff
@@ -0,0 +1,54 @@
+import pygame
+import unittest
+from unittest.mock import MagicMock
+from button import Button
+
+class TestButton(unittest.TestCase):
+    def setUp(self) -> None:
+        pygame.font.init()
+        self.test_button = Button("test", 250, 250)
+
+    def test_button_constructor(self):
+        with self.assertRaises(TypeError):
```

Comment:
Nice unit test, I didn't know about this syntax!

Reply:
I found it on their doc site.